src/enclave/rpc_sessions.h

```diff
@@ -48,6 +48,13 @@ namespace enclave
       to_host = writer_factory.create_writer_to_outside();
     }
 
+    void set_max_open_sessions(size_t n)
+    {
+      max_open_sessions = n;
+
+      LOG_INFO_FMT("Setting max open sessions to {}", n);
```

Comment:
should this be debug level?

Reply:
No - this is set by the host (so non-confidential), but happens after construction (so useful to log when it _has_ happened).